src/account.js

```diff
@@ -47,15 +47,16 @@ export const getAccountAddress = async (accountName) => {
       [name, t.String],
       [managerAddress, t.Address],
     ];
-    accountAddress = await executeScript({
+
+    const [result] = await executeScript({
       code,
       args,
       service: true,
     });
+    accountAddress = result;
   } catch (e) {
```

Comment:
I believe, we don't need to catch error here, since it will be caught inside `executeScript`

Reply:
You're right, changed!